pkg/operator/ceph/disruption/nodedrain/reconcile.go

```diff
@@ -209,7 +214,11 @@ func (r *ReconcileNode) reconcile(request reconcile.Request) (reconcile.Result,
 		if err != nil {
 			return reconcile.Result{}, errors.Wrapf(err, "node reconcile failed on op %s", op)
 		}
-		logger.Debugf("deployment successfully reconciled for node %s. operation: %s", request.Name, op)
+		if op == controllerutil.OperationResultCreated {
+			logger.Infof("deployment successfully reconciled for node %s. operation: %s", request.Name, op)
```

Comment:
nit: to reduce duplication you could format the message to a string var, then pass it to `Infof()` or `Debugf()`

Reply:
will do